Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -776,22 +778,31 @@ function vtkOpenGLTexture(publicAPI, model) {
     // We get the 6 images
     for (let i = 0; i < 6; i++) {
       // For each mipmap level
-      for (let j = 0; j <= model.maxLevel; j++) {
-        if (invertedData[6 * j + i]) {
-          const w = model.width / 2 ** j;
-          const h = model.height / 2 ** j;
-          model.context.texImage2D(
-            model.context.TEXTURE_CUBE_MAP_POSITIVE_X + i,
-            j,
-            model.internalFormat,
-            w,
-            h,
-            0,
-            model.format,
-            model.openGLDataType,
-            invertedData[6 * j + i]
-          );
+      let j = 0;
+      let w = model.width / 2 ** j;
+      let h = model.height / 2 ** j;
+      while (w >= 1 && h >= 1) {
```

Comment:
Should we exit if we are in WebGL2 and j reached maxLevel in order to avoid unnecessarily `texImage2D` calls?